Formula/uriparser.rb

```diff
@@ -35,13 +35,9 @@ def install
       system "cmake", "."
       system "make"
     end
-    ENV["GTEST_CFLAGS"] = "-I./gtest/googletest/include"
-    ENV["GTEST_LIBS"] = "-L./gtest/googletest/ -lgtest"
-    system "./autogen.sh" if build.head?
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          "--disable-doc"
-    system "make", "check"
+    system "cmake", ".", "-DGTEST_ROOT=#{buildpath}/gtest/googletest", "-DURIPARSER_BUILD_DOCS=OFF", *std_cmake_args
+    system "make"
+    system "make", "test"
```

Comment:
Is this needed to compile the program?

Reply:
What specific part are you questioning?

Reply:
Make test

Reply:
The previous iteration ran the tests, and I thought it was Homebrew policy to run tests?